### ISSUE TITLE
Make possible to disable repeated annoying notifications

### ISSUE THREAD

**Issue by pasichnichenko** [nextcloud/android#3855]
When I am using android app to sign in into remote nextclowd server and server is of outdated/unsupported version, app always pops huge 1/4 screen notification about server software is outdated. 

There must be checkbox with some label like "OK I know don't remind anymore" to acknowledge app I have read this warning and there are no need to annoy me In the future with this notification. 

Because nextclowd server is not always under end-user control and there are no reliable migration procedure to safely migrate with all files, contacts, notes, settings etc from old version to new one quickly and seamlessly. Sometimes there is an unavoidable need to use outdated server software for some certain period of time and notifications about expired server version are highly annoying, especially when you hit into bug of constant redirects back to login screen after few seconds of timeout in app or hitting anything in app.

This is concerning mobile application, android on my case.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky what do you think? We could have a similar mechanism we have for the new media folder notifications while I would say we should remeber the notification being shown per lowest supported server version as in, if we recommend 14+ and support 12+ we should store having shown it for 12+ and whenever we move to 13+ we show it again.

@pasichnichenko beware that we show this for a reason meaning also the client drops supporting server version older than 4 mayor versions compared to the latest mayor release, so if you still run Nc 11 the next client release is going to not let you use the app anymore!

**Comment by pasichnichenko** [comment]
I understand why you show this warning and it's OK, according to your policy and dropping anciently versioned serverd. But it is highly desirable to have some option to ack about version issue and stop showing notification again and again till reinstall of the app or reset app settings or complete inability to work with server.

**Comment by AndyScherzinger** [comment]
Yes, we should add that part and also only show it once per server version 👍
